lib/SPIRV/SPIRVRegularizeLLVM.cpp

```diff
@@ -293,6 +296,11 @@ void SPIRVRegularizeLLVMBase::lowerUMulWithOverflow(
   UMulIntrinsic->setCalledFunction(UMulFunc);
 }
 
+void SPIRVRegularizeLLVMBase::lowerBSwap(CallInst *BSwapIntrinsic) {
+  IntrinsicLowering IL(M->getDataLayout());
+  IL.LowerIntrinsicCall(BSwapIntrinsic);
```

Comment:
```suggestion
void SPIRVRegularizeLLVMBase::lowerBSwap(const std::vector<CallInst *> &BSwapInsts) {
  IntrinsicLowering IL(M->getDataLayout());
  for (auto *CI : BSwapInsts)
    IL.LowerIntrinsicCall(CI);
```

This variant less portable maybe, but it reduces number of recreations of IntrinsicLowering object.

Reply:
Or another variant:
```suggestion
void SPIRVRegularizeLLVMBase::lowerIntrinsic(CallInst *Intrinsic) {
  IntrinsicLowering IL(M->getDataLayout());
  IL.LowerIntrinsicCall(Intrinsic);
```
In this case we may use this method for lowering other intrinsics as well.

Reply:
I'd be very cautious calling this function for all intrinsics, because we don't control how intrinsics are lowered by this function and thus they may be lowered not in the way we want.

Reply:
I didn't mean to call it blind for all intrinsics in input module blind. Just to reuse it in the future for some another intrinsic that should be lowered and has lowering code in LowerIntrinsicCall function.

Reply:
Oh, I misinterpret you comment. Yes we can do that.

Reply:
> 

I'd prefer moving `IntrinsicLowering` constructor out of the loop too.

Reply:
I applied both suggestions in 3eb9828a1e1fa38b1d6f849381adfc92d35bd2cf